### ISSUE TITLE
Ran PENet pre-trained model and results do not match Kitti benchmark depth completion page

### ISSUE THREAD

**Issue by manifischer** [JUGGHM/PENet_ICRA2021#31]
Thank you very much for a very interesting paper.
I have run the PENet pre-trained model you've provided in evaluation mode on the cropped image.
in the results the code provided (val.csv under results) I got RMSE=757.197 MAE=209.001, compared to  RMSE=730.08	MAE=210.55 as it appears in the Kitti benchmark page.
IIs there a different PENet model that matches the submitted results? or there is something in the parameters that I put wrong (I kept the parameters as is in this repository).
Thanks a lot,
Mani

**Comment by JUGGHM** [comment]
Thanks for your interest! It's the same model as it is benchmarked on _**test**_ set while validated on **_val_** set. You could also refer to [this issue](https://github.com/JUGGHM/PENet_ICRA2021/issues/23).

**Comment by manifischer** [comment]
Thanks a lot, missed the issue you referred to

**Comment by kingLCH** [comment]
how to use penet pre-trained model,why it name is end with .tar
it seems cant be loaded by "checkpoint = torch.load(args.evaluate, map_location=device)"

**Comment by JUGGHM** [comment]
> how to use penet pre-trained model,why it name is end with .tar it seems cant be loaded by "checkpoint = torch.load(args.evaluate, map_location=device)"

Thanks for your interest! I am afraid taht it is an inherited bug and you can fix it yourself.
